### ISSUE TITLE
Uncaught TypeError: crypto.randomUUID is not a function

### ISSUE THREAD

**Issue by k33g** [Dunks1980/bay.js#3]
Hello, 
I get this error when loading my web application:
<img width="873" alt="image" src="https://user-images.githubusercontent.com/491848/212531088-62a544f1-43b6-4363-b471-a7369ec98e7c.png">

This is my `index.html` page:
```html
<html>
  <head>
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1">
  </head>
	<body>

    <template id="my-template">
      <h2>${this.message}</h2>
    </template>

    <div>
      <my-component bay="#my-template" message="Hello world!"></my-component>
    </div>

    <script type="module" src="main.js"></script>

	</body>
</html>
```

`main.js`:

```javascript
import bay from '//unpkg.com/@dunks1980/bay.js/bay.mjs';

bay();
```

I'm serving the page with: `python3 -m http.server` and I did the tests with Chrome (last version) and Firefox (last version) on macOS.

It looks like the cause comes from browser API restrictions: https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid

If I change the `makeid` function like this, everything seems fine:

```javascript
  function makeid(length) {

    let uuidv4 = () => {
      return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
        (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
      );
    }
    return uuidv4().replaceAll("-", "").substring(0, length);
  }
```

**Comment by Dunks1980** [comment]
@k33g , thankyou, I will take a look, and release a fix asap.

**Comment by Dunks1980** [comment]
@k33g , I have released fix in version 2.2.10

**Comment by k33g** [comment]
@Dunks1980 it works 🙂
